account_fiscal_position_partner_type/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,11 @@
+User can configure the default value according to the system company going to:
+   * Settings/Users & Companies/Companies.
+   * Changing the value of the "Default Partner Fiscal Position Type" field.
+
+Create a new partner and define the "Fiscal Position Type" field.
```

Comment:
I mentioned only the first part to be in `CONFIGURE.rst`. From here it should still go to `USAGE.rst`